libs/design/src/core/openable/openable.ts

```diff
@@ -1,17 +1,16 @@
 /**
  * An interface for giving a component the ability to display an open UI.
- * In order to be openable, the class must implement this property.
  */
 export interface DaffOpenable {
-  /** Whether the component is open or not */
+  /** Whether a component is open or not */
   open: boolean;
 
-  /** Sets reveal on the component */
+  /** Reveal the component */
   reveal: () => void;
 
-  /** Sets hide on the component */
+  /** Hide the component */
   hide: () => void;
 
-  /** Sets toggle on the component */
+  /** Toggles the component from open to not open */
```

Comment:
This will also work in reverse from not open to open, correct? The comment should reflect that converse behavior.

Reply:
updated